backend/battles/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+from django.urls import reverse
+
+from users.models import User
+
+
+class Battle(models.Model):
+    creator = models.ForeignKey(User, on_delete=models.CASCADE, related_name='battle_creator', )
```

Comment:
When choosing the `related_name` you need to consider:
1. It is going to be used from the related object
2. What kind of objects are going to be returned

In this situation, the related name is going to be used from an `user` object and it will return a list of battles. So the code would look like this:
```battles = user.battle_creator.all()```

Bit weird, right? IMHO something like this would look better:
```battles = user.battles_as_creator.all()```

Reply:
That looks really better! Thanks!